### DISCUSSION TITLE
Find the odd one

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#817] [Galaxy Trivia] [answered]
**EINMOACD**
**ITEMORINART**
**PROMNEFCARE**
**SUNTAPD**

**Comment by ParanoidUser** [comment]
@Yell0wflash / @angelabauer, any ideas? 😶‍🌫️

**Reply by Yell0wflash** [reply]
😊 ITEMORINART

**Comment by peterbaccaro** [comment, marked as answer]
1. COMEDIAN
2. TERMINATOR
3. PERFORMANCE
4. STANDUP

The odd one out in the given list is "**TERMINATOR**".

"COMEDIAN", "PERFORMANCE", and "STANDUP" are all related to the field of entertainment or live shows, specifically comedy. They are terms associated with comedians and their performances.
